lib/activities/get_extent.js

```diff
@@ -1,22 +1,27 @@
 "use strict";
 
-var url = require("url"),
+var debug = require("debug"),
+    url = require("url"),
     util = require("util");
 
 var shell = require("./shell");
 
+var log = debug("swfr:shell");
+
 /**
  * Fetch the extent of a raster.
  *
  * @returns [upper left, lower right]
  */
 module.exports = function getExtent(uri, callback) {
-  uri = url.parse(uri);
-
   var args = [
-    util.format("http://s3.amazonaws.com/%s%s", uri.hostname, uri.pathname)
+    "-nofl", // In case this is a VRT with too many files to be caught in stdout
+    uri
   ];
 
+  // Regex to extract coordinates.
+  var re = /([\[\(])([^,]*),(.*?)([\]\)])/;
```

Comment:
The code to extract coordinates wasn't working for me, and I couldn't figure out why, so I rewrote it.